query-engine/core/src/schema/query_schema.rs

```diff
@@ -434,6 +438,7 @@ pub enum ScalarType {
     Enum(EnumTypeRef),
     DateTime,
     Json,
+    JsonList,
```

Comment:
Why did we add this scalar type? We now have two ways to express a json list. Once as `ScalarType::List` and once as `OutputType::List(OutputType::Scalar)`

Reply:
`ScalarType::List` doesn't allow lists with multiple types of argument, until the next graphql standard is out that should allow that. We have here a json list with anything inside. So now we should be able to pass a list, such as `[1, bool, "foo"]` and that is completely valid.

And why JSON list is we want to force a vector out of the input, not just any kind of JSON object. If just using JSON and then just parsing a vector always, the input type would not be that sound anymore.